config/manager/manager.yaml

```
@@ -33,6 +33,8 @@ spec:
         env:
         - name: ANSIBLE_GATHERING
           value: explicit
+        - name: ANSIBLE_DEBUG_LOGS
```

I think the env var for that doesn't have the `_LOGS` suffix

```suggestion
        - name: ANSIBLE_DEBUG
```

https://docs.ansible.com/ansible/latest/reference_appendices/config.html#envvar-ANSIBLE_DEBUG

---

This is an operator-sdk variable, not an ansible one. https://sdk.operatorframework.io/docs/building-operators/ansible/development-tips/#viewing-the-ansible-logs

---

@dsavineau this is actually an ansible-operator-sdk env var, not an ansible-core specific one.  

ref: https://sdk.operatorframework.io/docs/building-operators/ansible/development-tips/#viewing-the-ansible-logs